LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2019 ReviewNB
+Copyright (c) 2019 - 2020 ReviewNB
```

Comment:
Do we really need an end date on a copyright? I believe the single year indicates the first year in which the work was published and covers the copyright then onwards.

My issue with this is -- what if we forget to update the notice next year? Do we loose copyright for work done in 2021 until this is updated.

Reply:
What I have studied so far is Copyright should be up to the date of publishing.

So, if it's a static content (such as the Times article you linked to), it should probably be statically copyrighted.

If it's **dynamically** generated content, it should be copyrighted to the current year. 

Another Approach to this is what used by Facebook([Prophet License](https://github.com/facebook/prophet/blob/master/LICENSE)). They don't mention Dates in the license.

Reply:
Quoting from https://opensource.stackexchange.com/questions/1522/what-should-be-written-in-mit-license-year-full-name

> Some people like to put in a range of years (i.e. 2015-2020) - this isn't necessary.


I also spoke to our lawyer and a single year is enough to enforce our copyright.  So I'm closing this PR. Thank you for thinking about it though.

Reply:
OK, I'll keep that in mind. Kudos to you for such an amazing project.